src/explore-education-statistics-common/src/modules/charts/util/__tests__/isOrphanedDataSet.test.ts

```diff
@@ -0,0 +1,231 @@
+          totalValue: '',
+          hint: 'Filter by school type',
+          legend: 'School type',
+          options: {
+            Default: {
+              label: 'Default',
+              options: [
+                {
+                  label: 'State-funded secondary',
+                  value: 'state-funded-secondary',
+                },
+                {
+                  label: 'State-funded primary',
+                  value: 'state-funded-primary',
+                },
+              ],
+            },
+          },
+          name: 'school_type',
+        },
+      },
+      footnotes: [],
+      indicators: [
+        {
+          label: 'Number of authorised absence sessions',
+          unit: '',
+          value: 'authorised-absence-sessions',
+          name: 'sess_authorised',
+        },
+        {
+          label: 'Number of overall absence sessions',
+          unit: '',
+          value: 'overall-absence-sessions',
+          name: 'sess_overall',
+        },
+        {
+          label: 'Authorised absence rate',
+          unit: '%',
+          value: 'authorised-absence-rate',
+          name: 'sess_authorised_percent',
+        },
+      ],
+      locations: [
+        { level: 'localAuthority', label: 'Barnsley', value: 'E08000016' },
+        { level: 'localAuthority', label: 'Barnet', value: 'E09000003' },
+      ],
+      boundaryLevels: [
+        {
+          id: 2,
+          label:
+            'Counties and Unitary Authorities December 2018 Boundaries EW BUC',
+        },
+      ],
+      publicationName: 'Pupil absence in schools in England',
+      subjectName: 'Absence by characteristic',
+      timePeriodRange: [
+        { code: 'AY', label: '2013/14', year: 2013 },
+        { code: 'AY', label: '2014/15', year: 2014 },
+        { code: 'AY', label: '2015/16', year: 2015 },
+      ],
+      geoJsonAvailable: true,
+    },
+    results: [],
+  };
+
+  const testFullTable = mapFullTable(testTableData);
+
+  test('is false for a data set that has matching location', () => {
+    const testDataSet: DataSet = {
+      filters: ['ethnicity-major-chinese', 'state-funded-primary'],
+      indicator: 'authorised-absence-sessions',
+      location: {
+        value: 'E08000016',
+        level: 'localAuthority',
+      },
+    };
+
+    expect(isOrphanedDataSet(testDataSet, testFullTable.subjectMeta)).toBe(
+      false,
+    );
+  });
+
+  test('is true for a data set where the location does not match on `value`', () => {
+    const testDataSet: DataSet = {
+      filters: ['ethnicity-major-chinese', 'state-funded-primary'],
+      indicator: 'authorised-absence-sessions',
+      location: {
+        value: 'does not exist',
+        level: 'localAuthority',
+      },
+    };
+
+    expect(isOrphanedDataSet(testDataSet, testFullTable.subjectMeta)).toBe(
+      true,
+    );
+  });
+
+  test('is true for a data set where the location does not match on `level`', () => {
+    const testDataSet: DataSet = {
```

Comment:
Just wondering again if you meant to put `${level}` instead of level with backticks ? If you didn't then mean to I think it's safe to remove the backticks

Reply:
There's no interpolation intended here, it's just referring to the names of the `value` and `level` fields by name. I've removed the backticks to avoid confusion. Thanks.